portfolio/src/main/webapp/calendar.js

```diff
@@ -0,0 +1,82 @@
+/** 
+ * Models a calendar event that is displayed on the UI.
+ * This class is useful when the event information needs to be sent 
+ * to the calendar servlet through a POST request. 
+ * This class contains all the event information that can be easily
+ * converted into a JSON string. 
+ * All fields are required. 
+ */
+class CalendarEvent {
```

Comment:
Indicate in the class method the types of the inputs.

Reply:
Updated.